libs/graphql/src/index.ts

```diff
@@ -14,3 +14,5 @@ export * from './__generated__/Orders';
 export * from './__generated__/OrderSub';
 export * from './__generated__/PartyAssetsQuery';
 export * from './__generated__/ProposalsQuery';
+
+export * from './data-providers';
```

Comment:
**thought (non-blocking)**:  I've finally got to answering in a slack thread about data-providers as an abstraction and I'm not a fan - I think the queries are better as close to the components that require the data as possible (https://www.apollographql.com/docs/react/data/operation-best-practices/#query-only-the-data-you-need-where-you-need-it). I'm going to mark this as non-blocking because the contents of this file can still be moved elsewhere before we go too far down the path of generic providers without everyone agreeing/agreeing that the correct balances are